src/components/FormRadioGroup/FormRadioGroup.tsx

```diff
@@ -40,7 +44,7 @@ export const FormRadioGroup = ({
   };
 
   return (
-    <div>
+    <div className={classes.centered}>
```

Comment:
Meanwhile we do not decide to change out style base tools can we keep using the components that material-ui offer to us? for example, this is equivalent to use `Box` thats creates a `div` by default but an be overried if you wish to another semantic component like `span`, more info here: https://react-hook-form.com/api/useformstate/

`import { Box } from '@material-ui/core`

`<Box justifyContent="center">`